### ISSUE TITLE
Component of pixel cannot be set individually using path expression

### ISSUE THREAD

**Issue by meijeru** [red/red#3729]
**Describe the bug**
An individual pixel (component of an image!) is a 4-tuple RGBA. Any tuple component can be retrieved, but setting it does not work.

**To reproduce**
```
>> i: make image! 1x1
> i/1
== 255.255.255.0
>> i/1/1
== 255
>> i/1/1: 0
== 0
>> i/1
==  255.255.255.0
```
**Expected behavior**
After the last setting, `i/1` should be `0.255.255.0`.

**Platform version (please complete the following information)**
```
Presumably all
```

**Comment by hiiamboris** [comment]
Yeah I'd like to see that working too. But FYI, it doesn't work in R2 at all, and R3 always changes the 4th component (alpha), regardless of the requested index.

**Comment by meijeru** [comment]
I can see no reason why it should not work in Red. I know that `poke` does not work on tuples (or any other direct value that has components, i.e. time date and pair) but path expressions do.

**Comment by Oldes** [comment]
Is there any good usage example for it?

**Comment by greggirwin** [comment]
The interesting thing is that there is a `poke` function in %tuple.reds, and `eval-path` calls that, so I don't know why it's not set in the action handlers when the datatype is registered.
